### ISSUE TITLE
Negative pytorch_spl_loss

### ISSUE THREAD

**Issue by yuhu-dsp** [ssarfraz/SPL#6]
Excuse me, but the pytorch_spl_loss.py keeps make negative loss in my code. 
Wondering why this loss is negative. Could you please help to explain a little.

**Comment by ConstantinSeibold** [comment]
Hi,
For this method, we want to maximize the similarity between the row/column profiles. However, as we usually work with gradient descent methods such as SGD/Adam/etc. , we tend to minimize objective functions. A quick workaround was to minimize the negative of the similarity. This leads to a negative loss function which should increase in absolute value but should get increasingly more negative over the course of the training.
This is further elaborated in Section 3 of the paper.

Does this help you with your question?

**Comment by yuhu-dsp** [comment]
Thank you for your answer! It helps a lot.
